CHANGELOG.md

```diff
@@ -48,7 +48,7 @@ all the usual things to it, such as scaling, rotating, cropping, tinting, making
 physics body, etc.
 
 Transparent videos are also possible via the WebM file format. Providing the video file has was encoded with
-an alpha channel, and providing the browser supports WebM playback (not all of them do), then it willl render
+an alpha channel, and providing the browser supports WebM playback (not all of them do), then it will render
```

Comment:
I could be wrong, but I'm otherwise unfamiliar w/ `willl`

Reply:
It's just a typo. However, the Change Log is sacrosanct and I don't correct typos or spelling mistakes in it. Any other file is fair game, though.